src/components/AlertInbox/Popup.tsx

```diff
@@ -10,16 +10,35 @@ export interface PopupProps {
 
 function Popup(props: PopupProps): JSX.Element {
   const [show, setShow] = useState(false);
+  const wrapperRef = useRef<HTMLDivElement>(null);
 
   const handleClose = () => {
     setShow(false);
     props.onClose();
   };
 
   useEffect(() => {
+    console.log('here');
```

Comment:
Looks like you just need to get rid of this console log for the linter to run correctly.

Reply:
Looks good otherwise.